test/Microsoft.AspNetCore.Mvc.Core.Test/RedirectToActionResultTest.cs

```diff
@@ -88,7 +88,7 @@ public class RedirectToActionResultTest
         {
             // Arrange
             var expectedUrl = "/Home/SampleAction#test";
-            var expectedStatusCode = StatusCodes.Status302Found;
+            var expectedStatusCode = StatusCodes.Status307TemporaryRedirect;
```

Comment:
Is there a reason to change the existing test, rather than add a new test? I probably don't care much either way, just curious.